plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/textdocument/TextDocumentServiceHandler.kt

```diff
@@ -31,31 +32,38 @@ class TextDocumentServiceHandler(
         )
     }
 
+    fun executeIfRunning(project: Project, runnable: (AmazonQLanguageServer) ->  Unit) =
```

Comment:
lift up to AmazonQLspService

Reply:
https://github.com/aws/aws-toolkit-jetbrains/pull/5386/files#diff-3116ed37b099b8b596a4df25501096b86fbe8bd2bb6bc1b1b3fc97e7785a348cR176-R178